docs/package.json

```
@@ -4,16 +4,20 @@
   "description": "Yugabyte Docs",
   "main": "public/index.html",
   "scripts": {
-    "start": "run-p webpack-watch hugo-server",
+    "start": "run-p start-diagrams webpack-watch hugo-server",
     "start-production": "run-p webpack-watch hugo-server-production",
     "start-static": "run-p webpack-watch hugo-server-static",
-    "build": "rm -rf public && npm ci && run-s webpack-build hugo-build",
-    "debug-build": "rm -rf public && npm ci && run-s webpack-build hugo-build-with-metrics",
+    "build": "rm -rf public && npm ci && run-p start-diagrams hugo-base-build",
+    "hugo-base-build": "run-s webpack-build hugo-build stop-diagrams",
+    "debug-build": "rm -rf public && npm ci && run-p start-diagrams hugo-base-debug-build",
+    "hugo-base-debug-build": "run-s webpack-build hugo-build-with-metrics stop-diagrams",
     "hugo-build": "hugo -b https://docs.yugabyte.com",
     "hugo-build-with-metrics": "hugo --templateMetrics --templateMetricsHints -b https://docs.yugabyte.com",
     "hugo-server": "hugo server --bind 0.0.0.0 --baseURL ${YB_HUGO_BASE:-localhost}",
     "hugo-server-production": "hugo server --bind 0.0.0.0 --baseURL ${YB_HUGO_BASE:-localhost} --environment production",
     "hugo-server-static": "hugo server --renderStaticToDisk --bind 0.0.0.0 --baseURL ${YB_HUGO_BASE:-localhost}",
+    "start-diagrams": "npm install --no-audit --no-fund && java -jar node_modules/yb-rrdiagram/rrdiagram*.jar --server --debug",
+    "stop-diagrams": "curl --silent localhost:1314/shutdown",
```

```suggestion
    "stop-diagrams": "curl --silent http://localhost:1314/shutdown",
```

probably better this way

---

added